src/main/java/frc/robot/subsystems/swerve/SwerveName.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.swerve;
+
+public enum SwerveName {
+    SWERVE("Real");
```

Comment:
```suggestion
    SWERVE("");
```
now there is only one swerve maybe real maybe sim so dont need name